### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by Juanse7793**
In this PR I made the Basic Setup for RoR & I fulfilled the following requirements

## Project requirements

- Create a new app. Give it a meaningful name. :heavy_check_mark: 
- Initialize your project with Git. :heavy_check_mark: 
- Make sure that your project has Postgres :elephant: database set up. :nerd_face: 
- Build your project schema. :rocket: 
    - Create and run the necessary migration files. :heavy_check_mark: 
    - Table and column names should match the ERD diagram. :+1: 
        - note: photo for users table should be a link to an image :100: 
    - Foreign keys should be included. :heavy_check_mark: 
    - All columns that are foreign keys should have a corresponding index. :100:

### CODE REVIEW

File: db/migrate/20220705164522_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.text :bio
```
Comment (julie-ify) [inline db/migrate/20220705164522_create_posts.rb]: - [ ] There's no `bio` column in the posts table according to the ERD. Kindly remove it.

File: db/migrate/20220705164609_add_user_ref_to_comment.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
```
Comment (julie-ify) [inline db/migrate/20220705164609_add_user_ref_to_comment.rb]: - [ ] According to the ERD, the foreign key in the comments table references the `author` in the Users table. Hint 👇 

```suggestion
  def change
    add_reference :comments, :author, foreign_key: { to_table: :users }, null: false
  end
```

File: db/migrate/20220705164740_add_user_ref_to_post.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
```
Comment (julie-ify) [inline db/migrate/20220705164740_add_user_ref_to_post.rb]: - [ ] According to the ERD, the foreign key in the posts table references the `author` in the Users table. Hint 👇 

```suggestion
  def change
    add_reference :posts, :author, foreign_key: { to_table: :users }, null: false
  end
```

File: db/migrate/20220705164909_add_user_ref_to_like.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+  end
```
Comment (julie-ify) [inline db/migrate/20220705164909_add_user_ref_to_like.rb]: - [x] According to the ERD, one of the foreign keys in the likes table references the `author` and not `user` in the Users table. Hint 👇 

```suggestion
  def change
    add_reference :likes, :author, foreign_key: { to_table: :users }, null: false
  end
```

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Juanse7793,

Good job so far!

## Highlights! 
- No linter errors ✔️ 
- Descriptive pull request ✔️  

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220705164909_add_user_ref_to_like.rb (hunk 4 above)
Comment (Juanse7793) [reply db/migrate/20220705164909_add_user_ref_to_like.rb]: thank you for your help :100:

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Juanse7793, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**